Lesson34/feyza/lesson34-app/src/components/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import { useState, useContext, useEffect } from "react";
+import "./styles.css";
+import { UserContext } from "../userContext";
+
+export const UserDetailsForm = ({ incomingUsername }) => {
+  const [username, setUsername] = useState(incomingUsername);
+  const { dispatch } = useContext(UserContext);
+
```

Comment:
Make sure to also get the username value from context and safe it in the username state